modules/indexer/code/internal/util.go

```diff
@@ -30,3 +34,16 @@ func FilenameOfIndexerID(indexerID string) string {
 	}
 	return indexerID[index+1:]
 }
+
+func FilenameMatchIndexPos(content string) (int, int) {
```

Comment:
Docs!
I'm unable to guess what this method is doing from its name, return type and implementation.
Everything about this method seems weird:
I have no idea why you return the index of the `7th` newline.
It sounds like you have a clear fileformat in front of your eyes, but I don't.

Reply:
Also, why is the first return value always `0`?

Reply:
Whenever there is a filename-based fetch, it will return the first seven lines of the file (BTW, this is mimics the github behavior).  This is used by the page to render the code snippet associated with the search result. 

So the start value will be always be zero because it the code snippted always start at the begining of the file when the hit is based on filename.